cardInfoAPI-Level1.yaml

```diff
@@ -0,0 +1,1402 @@
+        In order to prevent API consumers from having to fulfill PCI DSS requirements, no full PAN or other data relevant for this regulation are included.
+        The API consumer is expected to have a non-card related identifier for the customer for whom data should be returned (e.g. card account number, bank account number).
+      tags:
+        - Cards
+      parameters:
+        - in: query
+          name: contract_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/ContractReference'
+          description: Retrieves all cards related to a contract. Value has to be a functional reference to the contract that can be known by the end customer.
+        - in: query
+          name: person_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/PersonReference'
+          description: Retrieves all cards related to an individual person/cardholder. Value has to be a functional reference to the person that can be known by the end customer.
+        - in: query
+          name: card_status
+          required: false
+          schema:
+            $ref: '#/components/schemas/CardStatus'
+          description: Retrieves all cards in a certain status. Only one status can be filtered at a time.
+        - in: query
+          name: card_account_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/CardAccountReference'
+          description: Retrieves all cards related to a card account (not bank account). Value has to be a functional reference to the account that can be known by the end customer.
+        - in: query
+          name: bank_account_reference
+          required: false
+          schema:
+            $ref: '#/components/schemas/BankAccountReference'
+          description: Retrieves all cards related to a bank account (not card account). Value has to be a functional reference to the account that can be known by the end customer and is usually an IBAN.
+        - $ref: '#/components/parameters/limit'
+        - $ref: '#/components/parameters/offset'
+        - $ref: '#/components/parameters/client_in_header'
+        - $ref: '#/components/parameters/correlation_in_header'
+        - $ref: '#/components/parameters/agent_in_header'
+      responses:
+        '200':
+          description: Payment cards were found which match the query parameters. If no cards matched the parameters or access rights are missing, an empty list is returned.
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/X-Correlation-ID'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  cardList:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/CardDetails'
+                  _links:
```

Comment:
Why is `_links` used? Looks like we have a mix between `application/json` and `application/hal+json` of the response content type. I think we should be clear about this. Either we go with `application/json` without `_links` and/or with `application/hal+json` with `_links`

Reply:
The specification is following the current SFTI standard for pagination: https://github.com/swissfintechinnovations/.github/wiki/Pagination 
We plan to keep it as is.